package.json

```diff
@@ -28,13 +29,15 @@
     "@cypress/webpack-preprocessor": "^5.9.1",
     "@typescript-eslint/eslint-plugin": "^4.28.4",
     "@typescript-eslint/parser": "^4.28.4",
-    "cypress": "^8.0.0",
     "eslint": "^7.31.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-cypress": "^2.11.3",
     "eslint-plugin-import": "^2.23.4",
     "ts-loader": "^9.2.3",
     "typescript": "^4.3.5",
     "webpack": "^5.46.0"
+  },
+  "dependencies": {
+    "cypress": "^8.0.0"
```

Comment:
This shouldn't be dependencies, should be dev-dependencies

Reply:
when i changed it to dev-dependencies the last class the tests would not run

Reply:
yes, it was something related with the linter, could you move to dev dependency and see if it fails again? If it does not fail, please commit that change, if it fails, then let us know to approve this PR :)